soccer/gameplay/plays/offense/BasicOffensiveScramble.py

```diff
@@ -0,0 +1,73 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+
+import standard_play
+import evaluation
+import situational_play_selection
+import tactics.coordinated_pass
+import skills.move
+import skills.capture
+
+class BasicOffensiveScramble(standard_play.StandardPlay):
+
+    _situationList = [
+        situational_play_selection.SituationalPlaySelector.Situation.OFFENSIVE_SCRAMBLE
+    ] # yapf: disable
+
+    class State(enum.Enum):
+        get_ball = 1, 'Start the play'
```

Comment:
You dont need a state. You can transition from start to running and capture in the running state and transition to completed from there